.circleci/config.yml

```diff
@@ -225,6 +228,29 @@ references:
         mkdir ./unit-tests
         demisto-sdk lint -p 8 -g -vvv --test-xml ./unit-tests --log-path ./artifacts --failure-report ./artifacts --coverage-report $ARTIFACTS_FOLDER/coverage_report --docker-image << parameters.dockerimageflag >>
 
+  run_lint_for_native_testing: &run_lint_for_native_testing
```

Comment:
can we reuse run_unit_testing_and_lint and refactor the different demisto-sdk args into parameters here?